saviSimulation/savi/jason_processing/Geometry.java

```diff
@@ -33,11 +33,7 @@
 		double azimuth = theta1 - refAngle;
 
 		// to normalize between 0 to 2 Pi                                                                
-		while (azimuth < 0)                                                                                    
-			azimuth += (2 * Math.PI);
-		
-		while (azimuth > (2 * Math.PI))
-			azimuth -= (2 * Math.PI);
+		azimuth = normalize02PI(azimuth);
```

Comment:
I think it may be better to access this method statically: 
azimuth = Geometry.normalize02PI(azimuth);